msticpy/data/queries/m365d/kql_m365_hunting.yaml

```diff
@@ -8,12 +8,13 @@ defaults:
   metadata:
     data_source: 'hunting_queries'
   parameters:
-      start:
-        description: Query start time
-        type: datetime
-      end:
-        description: Query end time
-        type: datetime
+      # The m365d hunting queries doesn't use start and end time param, time range is implemented statically.
```

Comment:
I think it would be cleaner to just delete these.
I know the feeling of reluctance to delete someone else's code but I think this is just a result of copy/paste of other files.